app/src/editor/canvas/components/ModuleHelp.jsx

```diff
@@ -22,23 +26,53 @@ export default class ModuleHelp extends React.Component {
     }
 
     async load() {
-        const { moduleId } = this.props
-        const help = await services.moduleHelp({
-            id: moduleId,
-        })
+        const { moduleId, moduleName } = this.props
+
+        const cleanedName = moduleName.replace(/\s/g, '').replace(/\(/g, '_').replace(/\)/g, '')
+
+        // eslint-disable-next-line global-require, import/no-dynamic-require
+        const helpJson = await require(`$newdocs/content/canvasModules/${cleanedName}-${moduleId}.jsx`)
+        const { help } = helpJson.default
+
+        // eslint-disable-next-line global-require, import/no-dynamic-require
+        const helpMdPath = await require(`$newdocs/content/canvasModules/${cleanedName}-${moduleId}.md`)
+        const helpMdText = await fetch(helpMdPath).then((res) => res.text())
+
         if (this.unmounted) { return }
         this.setState({
-            [moduleId]: help,
+            inputs: help.inputs,
+            inputsRaw: helpJson.default.inputs,
+            outputs: help.outputs,
+            outputsRaw: helpJson.default.outputs,
+            params: help.params,
+            paramsRaw: helpJson.default.params,
+            helpMdText,
         })
     }
 
     render() {
-        const { className, moduleId } = this.props
-        const help = this.state[moduleId] || {}
+        const { className } = this.props
+        const {
+            inputs,
+            inputsRaw,
+            outputs,
+            outputsRaw,
+            params,
+            paramsRaw,
+            helpMdText,
+        } = this.state
         return (
             <div className={cx(className)}>
-                {/* eslint-disable react/no-danger */}
-                <div dangerouslySetInnerHTML={{ __html: help.helpText }} />
+                <ReactMarkdown source={helpMdText} />
+                <CanvasModuleConfig
+                    moduleInputs={inputs || {}}
+                    moduleInputsRaw={inputsRaw || []}
+                    moduleOutputs={outputs || {}}
+                    moduleOutputsRaw={outputsRaw || []}
+                    moduleParams={params || {}}
+                    moduleParamsRaw={paramsRaw || []}
+                    moduleHelpAvailable={helpMdText && !helpMdText.includes(noHelpTxt)}
```

Comment:
Perhaps could apply these defaults when you init the state & at the point where you call `setState`, or as `defaultProps`/default values in `CanvasModuleConfig`? Minor.